### ISSUE TITLE
maybe not support cn dbcs well

### ISSUE THREAD

**Issue by qyh214** [stoneharry/Spell-Editor-GUI-V2#27]
I use your all of the dbcs run well but If I use cn spells after import spell to mysql,click one spell then you can see always
![qq 20160730213052](https://cloud.githubusercontent.com/assets/3896160/17270864/61f4de2e-569f-11e6-803f-a131ee468e6e.png)

so I put cn spell dbcs to you maybe you can solve it

I suspect that the structure of Chinese dbc file, you may need to check the file structure

  my cn spell link
https://mega.nz/#!M9ZRDa4a!nrY6Trj9r1X5wSFTNQiHD97JJswzOeBb7_PvswZuj7A

**Comment by stoneharry** [comment]
Thanks, will check it out when I get time.

**Comment by stoneharry** [comment]
The latest release might fix this. Can you check please? This is v1.5 as of this post.

**Comment by qyh214** [comment]
I have test!run well!thanks!

**Comment by qyh214** [comment]
but if I choose spellid 70938 it will become like 
![1](https://cloud.githubusercontent.com/assets/3896160/21352547/f2fe365e-c6fc-11e6-8f6a-cb04bc297011.png)

**Comment by stoneharry** [comment]
Confirmed broken loading 70938. It's because AreaGroup was being loaded as a Int32 and it needs to be unsigned. Spells like 70938 had a value of 4294967295 which overflows regular old Int32's.

Fixed in e901c55e0ae4340dd93fa85e4cc2e7edbc473681. I will post a release for this shortly (it will have to be v1.5.2 since I just did a release missing this. I'll maybe do this later in the week - compile from the repository to get it now).
